client/my-sites/domains/domain-management/name-servers/style.scss

```diff
@@ -5,12 +5,24 @@
 }
 
 .name-servers__dns {
-	.name-servers__toggle {
-		display: inline-block;
-		float: right;
+	display: flex;
```

Comment:
Do we really need the card to be `flex`?

Reply:
I have removed this also.